sdk/communication/azure-communication-callautomation/azure/communication/callautomation/aio/_call_connection_client_async.py

```diff
@@ -1010,15 +1011,15 @@ async def hold(
         *,
         play_source: Optional[Union['FileSource', 'TextSource', 'SsmlSource']] = None,
         operation_context: Optional[str] = None,
-        operation_callback_uri: Optional[str] = None,
+        operation_callback_url: Optional[str] = None,
```

Comment:
why was this renamed? Wouldn't this be a breaking change @kashifkhan?

Reply:
This is mistake - it should be ..._url, like other instances.
CallAutomation has branching policy where we merge master with latest features ("alpha features") and we have separate branch for release. It should be ok to make changes here, as this only changes in alpha